src/style.css

```diff
@@ -1 +1,35 @@
-/* add styles here */
+:root {
+  --stars-active: url(/src/images/star-active.svg);
+  --stars-inactive: url(/src/images/star.svg);
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background-image: var(--stars-inactive);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars .stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: var(--stars-active);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
you already specified these styles above
<img width="457" alt="image" src="https://github.com/mate-academy/layout_stars/assets/70220667/9c8a471d-0e85-4c52-86b7-8b2e8b17d1eb">
```suggestion
```

so better to remove it